### ISSUE TITLE
Who disagrees the most?

### ISSUE THREAD

**Issue by pasiegrist** [jsoma/data-studio-projects#158]
# Pitch

## Summary

The Swiss democracy is different. Lot of referendums are held every year. There is a broad range of topics the swiss can vote on: they banned minaretts, they repealed a minimum wage or smashed six weeks holiday per year at the ballot. The thing is: Switzerland is not so different to other countries when it comes to polarisation and the differences between progressive urban and conservative rural areas. Especially people from the cities often have the feeling, that they are on the losing side, when votes are held. Which means that their county voted differently, than the national outcome was. I wanted to see, if this is true. And: I want to find the county in Switzerland that votes the most different that the rest.
## Details

**Possible headline(s):** 
People from this county are not happy about how the Swiss vote

**Data set(s):** 

I obtain the data from the Swiss Federal Statistical Office

Https://www.bfs.admin.ch/bfs/de/home/statistiken/politik/abstimmungen.assetdetail.3362356.html

https://www.bfs.admin.ch/bfs/de/home/statistiken/regionalstatistik/regionale-portraets-kennzahlen/gemeinden.html

**Code repository:** 
https://github.com/pasiegrist/Datastudio/tree/master/Code/01-who_loses_the_most
	
**Possible problems/fears/questions:** 
Problem: I would like to visit the place and write about it – no can do. 
Nice to have? I would like to build a little javascript tool, that allows people to check the ratio for the county that they are living in.
Fears: I would like to do a small county portrait with data. I am not sure that this is going to workout. 

## Work so far

I have downloaded the CSV's from the database, cleaned them up and got them into pandas. I calculated the ratio and found the Top 5 who lose the most and the one that win the most. 

Here is a graph, showing the distribution of how many counties what ratio have.

<img width="637" alt="screen shot 2018-07-16 at 20 37 53" src="https://user-images.githubusercontent.com/15381149/42790357-a121ee16-8938-11e8-9bb9-9c3f766a79b6.png">

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by pasiegrist** [comment]
# Update
    	
## New graphs

<img width="883" alt="screen shot 2018-07-19 at 12 35 05" src="https://user-images.githubusercontent.com/15381149/42957300-3ede4596-8b50-11e8-92e8-28eee6c7f0ca.png"> 


<img width="539" alt="screen shot 2018-07-19 at 12 35 15" src="https://user-images.githubusercontent.com/15381149/42957298-3ed154bc-8b50-11e8-8aa4-18272b7cd691.png">




      
    ## Any changes in direction or topic?
      Not so far. 
    	
    ## Problems/Questions
      
The problem is, I couldn't find any significant pattern within counties. I looked for example for a relation between the share of settlement area and the win/loser-ratio – but there is none.  

<img width="552" alt="screen shot 2018-07-19 at 12 34 51" src="https://user-images.githubusercontent.com/15381149/42957310-441ea870-8b50-11e8-9ac1-ce73f9eda586.png">

And I really would like to write pieces about these counties – but this would take some classical reporting and visiting the places, which is not possible for now. I abandoned the Idea of making a javascript based search engine – but I still would love to have one.

    	
    ## Checklist
    	
    - [ ] I have included my visuals
    - [ ] I have filled out the sections above
    - [ ] I have been updating my DIARY.md with details about my process
    - [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Greetings! I'm a little robot, just having a little look around.

You need some feedback, let me summon @ella24, @nickospi, @mattrehbein for you

**Comment by nickospi** [comment]
- super interesting graph ideas and I like the fact that I can read them and understand them without having to study the background of your story.
- I would definitely sort both graphs in the same order eg. if Schelten is first in the first graph, it could be first in the second one too and maybe I would see this as an one complete infographic feat all the info in those 2 separate graphs.

**Comment by mattrehbein** [comment]
Speaking for the American audience, you couch the story in your pitch beautifully, from the polarization angle that is familiar to use and therefore makes me interested to read on about how it can be the same in other countries.

The bar graphs that break down how often the four individual counties voted with the winning side or not pay off your thesis the best for me. Does the chart show the two biggest winners and the two biggest losers? I wasn't entirely clear on that from the graph's title, and the top two do have lower percentages, but since they still are with the majority nearly 60% of the time, it made me question whether they are in fact the biggest losers.

**Comment by ella24** [comment]
Maybe is because I am not familiar with Switzerland at all, but I do not understand the first graph. Are the two at the top the ones that win the most or lose the most?
For the second graph I would use the same order of the counties from the first graph and explain a bit more what is happening. Please feel free to illustrate me more about what you are trying to communicate so I can give you better feedback.

**Comment by pasiegrist** [comment]
Thanks for the feedback Coral. I hope after second revision, it is going to get clearer for Non-Swiss :)

**Comment by pasiegrist** [comment]
# Update
	
## Your project content: images/words/etc

The Swiss democracy is different. A lot lot of referendums are held every year. There is a broad range of topics the Swiss can vote on: they banned minaretts, they repealed a minimum wage or smashed six weeks holiday per year – those are just a few examples of the 160 votes the Swiss took between 2000 and 2017.

The thing is: Switzerland is not so different to other countries when it comes to polarisation and the differences between progressive urban cities and conservative rural areas. Especially people from the cities often have the feeling, that they are on the losing side, when votes are held. Which means that their  municipality voted differently, than the national outcome was. I wanted to see how much this is true and got surprised.

I analyzed, how often a municipality voted with the winning side to find the ones, where this happens the least and the most. First of all the findings for the most were not the big surprises. It is Adliswil and Langenthal. Two municipalities that with their size and political landscape represent somewhat a Swiss average. Which means when elections are held, the right-wing populist party will get most of the votes followed by either the social democratic or the liberal party. On average, a Swiss municipality votes in 84.13 Percent with the winning side. 

**These municipalities voted the most with the winning side**
![size_winners](https://user-images.githubusercontent.com/15381149/43162971-acd804b6-8f5a-11e8-9806-b5c281f4db83.png)

The ones who lose the most, still vot almost 60 Percent of the time with the winning side. 

**These municipalities voted the least with the winning side**
![size_losers](https://user-images.githubusercontent.com/15381149/43162929-8964d0a4-8f5a-11e8-81cf-31b246a50de8.png)




## Any changes in direction or topic?

Not so far.
	
## Problems/Questions
  
Question: As mentioned in Update I looked for patterns, but couldn't find one. Or do I? Maybe I should do a log on the y-scale?


## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by xeophin** [comment]
Hi, I have been summoned by a bot.

I like the idea of checking out what municipality follows the general vote most closely. Might be interesting to watch these municipalities in the future, to get a glimpse how the votes are being cast?

Some feedback:
- I'd add a bit more space between each municipality, so that it forms a neat block that's separate from the next one
- I see why you would want to use a limited colour palette, but it confused me at first, since I connected the winning/losing ratio to the settlement area and didn't see how one value "converts" to the other. Might be good to use different colours for different things
- For both the settlement area and population charts you would need to give a scale at the top – otherwise it just seems like a random square
- I like the coat of arms, but I would remove the gradient (yes, I know that on Wikipedia they add that everywhere ... but I think it looks a bit tacky).
- Datawise: I think the BfS has a classification for municipalities, putting them in urban, peri-urban and ... rural(?) categories, maybe that might be worth exploring. It is definitely interesting that you wouldn't get big (western) cities like Lausanne or Geneva as the big losers here.

**Comment by pasiegrist** [comment]
Thanks a lot Kaspar. I have tried to adapt your design critique. 

<img width="418" alt="screen shot 2018-07-26 at 18 22 31" src="https://user-images.githubusercontent.com/15381149/43291955-7bba4e90-9101-11e8-904c-40a7846925f7.png">

**Comment by pasiegrist** [comment]
# Final

## Project visuals/text

The Swiss democracy is different. A lot lot of referendums are held every year. There is a broad range of topics the Swiss can vote on: they banned minaretts, they repealed a minimum wage or smashed six weeks holiday per year – those are just a few examples of the 160 votes the Swiss took between 2000 and 2017.

The thing is: Switzerland is not so different to other countries when it comes to polarisation and the differences between progressive urban cities and conservative rural areas. Especially people from the cities often have the feeling, that they are on the losing side, when votes are held. Which means that their municipality voted differently, than the national outcome was. I wanted to see how much this is true and got surprised.

**Introducing: The Winners**

I analyzed, how often a municipality voted with the winning side to find the ones, where this happens the least and the most. First of all the findings for the most were not the big surprises. It is Adliswil and Langenthal. Both municipalities represent with their size and political landscape somewhat a Swiss average. Which means when elections are held, the right-wing populist party will get most of the votes followed by either the social democratic or the liberal party.

**These municipialities are the winners**
_Share of votes they voted with the winning side_
![winners_size_b](https://user-images.githubusercontent.com/15381149/43344660-91863ba0-91b8-11e8-9075-cf73993ce8b2.png)

**Introducing: The Losers**

Now let's talk about the losers. It was kind of surprising, that not one of the western cities like Lausanne or Geneva showed up at the end of the list as they are traditionally leftist, which Switzerland is not. With a share of 74 percent (Lausanne) and 72 percent (Geneva) they are below the average of around 84 percent, but they are not at the far end. 

The two municipalities that vote the least along with the overall outcome are two small towns, one in the canton of Jura called Schelten and the other one is Campo (Vallemaggia) in Ticino. They vote differently in more than every third vote.

**These municipialities are the losers**
_Share of votes they voted with the winning side_

![losers_size_b](https://user-images.githubusercontent.com/15381149/43344719-bcf80066-91b8-11e8-936a-7deaaa1d7f8a.png)


## Details

**Headline:** 
Nobody loses as often as those two

**Published website version:**
https://pasiegrist.github.io/

**Code repository:**
https://github.com/pasiegrist/Datastudio/tree/master/Code/01-who_loses_the_most

**Final data set(s):** 
https://github.com/pasiegrist/Datastudio/tree/master/Code/01-who_loses_the_most

## What did you find to be the most difficult part of this project?
The most difficult was translating the story into a graphics.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Nope, I am not. I could not find a nice pattern and I wasn't able to produce a Javascript base search-tool, where people could check for the result of the municipality.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
